src/forms/CurrentIncome.js

```diff
@@ -44,7 +44,7 @@ import { getTimeSetter } from '../utils/getTimeSetter';
 *
 * @returns Component
 */
-const IncomeForm = function ({ current, time, setClientProperty }) {
+const IncomeForm = function ({ current, time, setClientProperty, snippets }) {
 
   var type = 'income';
```

Comment:
Just a note, we're going to change this once we've decided how to provide the standard "Income Type" and "Expense Type" headers to all the pages that need them.